src/pages/ListPage/MiniSlider.js

```diff
@@ -0,0 +1,241 @@
+import React, { Component, useEffect, useState } from 'react';
+import Slider from 'react-slick';
+import styled from 'styled-components';
+import theme from '../../styles/theme';
+import 'slick-carousel/slick/slick.css';
+import 'slick-carousel/slick/slick-theme.css';
+import movieIcon from './images/movie.png';
+import { AiOutlineRight } from 'react-icons/ai';
+import rightArrow from './images/arrow-right.png';
+
+const MiniSlider = () => {
+  //부모 컴포넌트에서 받은 state와 method
+
+  const [product, setProduct] = useState([]);
+
+  useEffect(() => {
+    fetch('/data/category.json')
+      .then((result) => result.json())
+      .then((data) => setProduct(data));
+  }, []);
+
+  console.log(product);
+
+  function SampleNextArrow(props) {
+    const { className, style, onClick } = props;
+    return (
+      <div
+        className={className}
+        style={{
+          ...style,
+          display: 'block',
+          background: 'gray',
+          width: '40px',
+          height: '40px',
+        }}
+        onClick={onClick}
+      />
+    );
+  }
+
+  function SamplePrevArrow(props) {
+    const { className, style, onClick } = props;
+    return (
+      <div
+        className={className}
+        style={{
+          ...style,
+          display: 'block',
+          background: 'gray',
+          width: '40px',
+          height: '40px',
+        }}
+        onClick={onClick}
+      />
+    );
+  }
+
+  const settings = {
+    dots: true,
+    infinite: false,
+    speed: 1000,
+    slidesToShow: 4,
+    slidesToScroll: 4,
+    initialSlide: 4,
+    nextArrow: (
+      <SampleNextArrow>
+        <img src={rightArrow} alt={'arrow'} />
+      </SampleNextArrow>
+    ),
+    prevArrow: <SamplePrevArrow />,
+    responsive: [
+      {
+        breakpoint: 1024,
+        settings: {
+          slidesToShow: 3,
+          slidesToScroll: 3,
+          infinite: true,
+          dots: true,
+        },
+      },
+      {
+        breakpoint: 600,
+        settings: {
+          slidesToShow: 2,
+          slidesToScroll: 2,
+          initialSlide: 2,
+        },
+      },
+      {
+        breakpoint: 480,
+        settings: {
+          slidesToShow: 1,
+          slidesToScroll: 1,
+        },
+      },
+    ],
+  };
+
+  return (
+    <>
+      <SliderTilte>
+        <img src={movieIcon} alt={`movie`} />
```

Comment:
```suggestion
        <img src={movieIcon} alt="movie" />
```

딱히 변수도 아니고 {}를 쓸 필요는 없어보이네요